Luther unable to create prs

### What happened?

https://github.com/vybestack/llxprt-code/actions/runs/20186769334

Run set -euo pipefail
Selection mode: manual
Branch name: luther-issue652
Checking if PR exists for branch luther-issue652...
✗ No existing PR found for branch luther-issue652, creating new one
HTTP 401: Bad credentials (https://api.github.com/graphql)
Try authenticating with:  gh auth login
Error: Process completed with exit code 1.

### What did you expect to happen?

Create the pr

### Client information

<details>

```console
$ llxprt /about
# paste output here
```

</details>


### Login information

_No response_

### Anything else we need to know?

_No response_

--- a/.github/workflows/luther.yml
+++ b/.github/workflows/luther.yml
@@ -613,8 +613,8 @@ jobs:
         id: pr
         if: steps.commit_push.outputs.has_changes == 'true'
         env:
-          GH_TOKEN: ${{ secrets.GH_SECRET }}
-          GITHUB_TOKEN: '' # Override job-level GITHUB_TOKEN to force gh to use GH_TOKEN
+          GH_TOKEN: ${{ github.token }}
+          GITHUB_TOKEN: ${{ github.token }}
           ISSUE_BRANCH: ${{ steps.select_issue.outputs.branch }}
           ISSUE_TITLE: ${{ steps.issue_data.outputs.title }}
           ISSUE_NUMBER: ${{ steps.select_issue.outputs.issue_number }}

